explore.py

```diff
@@ -29,6 +29,13 @@ def main(args):
         percentiles)
     for p, pv in zip(percentiles, percentile_values):
         print("%2.2f%% of customers spend less than: $%.2f" % (p, pv))
+    
+
+    # The fraction of customers that have non-zero revenue
+    print("\n")
```

Comment:
If you just meant to print one line, use `print()`. Otherwise, print automatically appends a newline to whatever you print, so this will print two lines. (https://www.geeksforgeeks.org/print-without-newline-python/)